documentation/design-docs/ipc-protocol.md

```diff
@@ -459,10 +499,9 @@ All errors will result in the Server closing the connection.
 
 Error response Messages will be sent when:
 * the client sends an improperly encoded Diagnostic IPC Message
-* the client uses an unknown `command_set`
-* the client uses an unknown `command_id`
+* the client uses an unknown `command`
```

Comment:
You should also condense it above in the error code enumeration.

Reply:
Good catch!  I did it in the implementation, but not here.